Makefile

```diff
@@ -130,8 +135,17 @@ clean-tests:
 #------------------------------------------------------
 # Run Targets
 #------------------------------------------------------
-unit-tests:
+# Envtest use for checking the deployment operator
+ENVTEST ?= $(GOBIN)/setup-envtest
+ENVTEST_K8S_VERSION = 1.28.0
+.PHONY: envtest
+envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
+$(ENVTEST): $(GOBIN)
+	test -s $(GOBIN)/setup-envtest || GOBIN=$(GOBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
```

Comment:
shouldn't this be set to `@$ENVTEST_K8S_VERSION` instead of `@latest`?

Reply:
the ENVTEST_K8S_VERSION (is the k8s version) that needs to match the setup-envtest.

Reply:
I can replace it with: v0.0.0-20240215124517-56159419231e
But I prefer to use it like the Kubebuilder: https://github.com/kubernetes-sigs/kubebuilder/blob/8afeb403549fa87bf55a00e10a819a34719eec53/testdata/project-v4/Makefile#L169